Firestore/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Unreleased
 
+# v1.11.3
+- [changed] Changed the default log level to `notice` instead of `warning` (#5196).
+- [added] The Firestore iOS SDK can now be built via CMake (#5052).
```

Comment:
This isn't supported. We have a CMake build to make it possible for us to combine this code with the C++ SDK, but that's an implementation detail of the C++ SDK, not a supported usage path for Apple platform users.

Reply:
Removed.